tests/correctness/exponent.futil

```diff
@@ -0,0 +1,30 @@
+import "primitives/std.lib";
+import "primitives/bitnum/math.futil";
+component main() -> () {
+  cells {
+    base = std_mem_d1(32, 1, 1);
+    exp = std_mem_d1(32, 1, 1);
+    out = std_mem_d1(32, 1, 1);
+    std_exp0 = std_exp();
+  }
+  wires {
+    group init {
+      base.addr0 = 1'd0;
+      exp.addr0 = 1'd0;
+      init[done] = 1'd1;
+    }
```

Comment:
I meant in the test here  @rachitnigam

Reply:
Yeah, this program is undefined. Using groups with done condition `1` within a seq doesn't work. `invoke` has a terrible interface—the inputs must be driven forever. This means that `init` should read the values those memories and saves them to registers. Next, you should use the registers to supply the value to the `invoke`.